site/links.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="gemi-nikk">
+    <meta name="description" content="A personal website featuring a bio, favorite things, a haiku, and useful links.">
+    <title>kishan's CSD-112 Project</title>
+    <link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+    <link rel="manifest" href="/site.webmanifest" />
+</head>
+<body>
+<div class="width-wrap">
+    <header id="top">
+        <nav>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="links.html">Links</a></li>
+                <li><a href="glossary.html">Glossary</a> </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section id="useful-links">
+            <h1>Useful Links That Are Help To Understand Stock Market And For Research</h1>
+            <ul>
+                <li><a href="https://www.moneycontrol.com/" target="_blank" rel="noopener noreferrer">Money Control</a></li>
+                <li><a href="https://stockedge.com/" target="_blank" rel="noopener noreferrer">Stockedge</a></li>
+            </ul>
```

Comment:
3. `Lab 2 Site has > links page with > <main> has > an unordered list with > a link to the HTML Validator`
4. `Lab 2 Site has > links page with > <main> has > an unordered list with > a link to the Favicon Generator`
```suggestion
            <ul>
                <li><a href="https://www.moneycontrol.com/" target="_blank" rel="noopener noreferrer">Money Control</a></li>
                <li><a href="https://stockedge.com/" target="_blank"  rel="noopener noreferrer">Stockedge</a></li>
                <li><a href="https://validator.w3.org/" target="_blank" title="HTML Validator">HTML Validator</a></li>
                <li><a href="https://realfavicongenerator.net/" target="_blank" title="Favicon Generator">Favicon Generator</a></li>
            </ul>
```